cms/djangoapps/contentstore/management/commands/add_instructor_ids.py

```diff
@@ -0,0 +1,102 @@
+"""
+Command to add UUID for Instructor.
+"""
+
+import uuid
+import logging
+
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand, CommandError
+
+from opaque_keys.edx.keys import CourseKey
+
+from openedx.core.djangoapps.models.course_details import CourseDetails
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+
+
+log = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Example usage:
+        ./manage.py cms add_instructor_ids --username <username> --orgs <org1> <org2> ...
+        --course_keys <key1> <key2> ... --settings=devstack
+    """
+    help = './manage.py cms add_instructor_ids --username <username> --orgs <org1> <org2> ... '\
+           '--course_keys <key1> <key2> ... --settings=devstack'
+
+    def add_arguments(self, parser):
+        """
+        Add arguments to the command parser.
+        """
+
+        parser.add_argument(
+            '--username',
+            required=True,
+            help='Enter an existing username',
+        )
+
+        parser.add_argument(
+            '--orgs',
+            nargs='+',
+            help='Enter one or more organizations space separated',
+            required=False,
+            default=[]
+        )
+
+        parser.add_argument(
+            '--course_keys',
+            nargs='+',
+            help='Enter one or more course keys space separated',
+            required=False,
+            default=[]
+        )
+
+    @staticmethod
+    def _populate_instructor_uuid(course_keys, username):
+        """
+        Populate the uuid in each instructor.
+        """
+        for key in course_keys:
+            key_object = CourseKey.from_string(key)
+            course_descriptor = CourseDetails.fetch(key_object)
+
+            # Adding UUID in each instructor
+            for instructor in course_descriptor.instructor_info.get("instructors", []):     # pylint: disable=E1101
+                if "uuid" not in instructor:
+                    instructor["uuid"] = str(uuid.uuid4())
+
+            # Updating the course
+            CourseDetails.update_from_json(
```

Comment:
Can we use an `updated` flag and only call `update_from_json` if we added a UUID above?